common/functions/tests/it/scalars/maths/exp.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_datavalues::prelude::*;
+use common_exception::Result;
+use common_functions::scalars::*;
+
+#[test]
+fn test_exp_function() -> Result<()> {
+    struct Test {
+        name: &'static str,
+        display: &'static str,
+        columns: DataColumn,
+        expect: DataColumn,
+        error: &'static str,
+        func: Box<dyn Function>,
```

Comment:
exp function is a stateless function, you should remove Test struct func to make the test code more clean, then all the test cases use a same ExpFunction

Reply:
recommend: remove the func field in Test struct

Reply:
Yes， we can share a single func field in all tests.